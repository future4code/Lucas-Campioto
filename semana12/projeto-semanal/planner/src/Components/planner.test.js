import React from 'react'
import renderer from "react-test-renderer";
import PlannerTasks from "./Planner";

describe('teste componente PlannerTasks', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(<PlannerTasks></PlannerTasks>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    })
})
