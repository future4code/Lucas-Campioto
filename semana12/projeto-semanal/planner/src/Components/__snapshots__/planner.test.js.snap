// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`teste componente PlannerTasks renders correctly 1`] = `
<div
  className="sc-bdVaJa eIffcY"
>
  <div>
    <div>
      <h3>
        Segunda
      </h3>
    </div>
  </div>
  <div>
    <div>
      <h3>
        Terça
      </h3>
    </div>
  </div>
  <div>
    <div>
      <h3>
        Quarta
      </h3>
    </div>
  </div>
  <div>
    <div>
      <h3>
        Quinta
      </h3>
    </div>
  </div>
  <div>
    <div>
      <h3>
        Sexta
      </h3>
    </div>
  </div>
  <div>
    <div>
      <h3>
        Sabádo
      </h3>
    </div>
  </div>
  <div>
    <div>
      <h3>
        Domingo
      </h3>
    </div>
  </div>
</div>
`;
